### PR TITLE
chore: contribute-update

### PR DESCRIPTION
**Pull request by jpvajda**
Updated contributor guide and adds a PR checklist the PR Template.

### CODE REVIEW

**Comment by aswanson-nr** [comment]
The repository has been renamed to `newrelic/newrelic-quickstarts`. Github will handle routing your remotes to this new url, but we recommend that you update your remote as soon as possible.
**NOTE**: As part of this change, we have moved all the quickstarts from the `/packs` directory to the `/quickstarts` directory. To bring your pull request up to date, please pull in the latest commits from the `main` branch.

Use the following command to set your new remote:
```bash
git remote set-url origin https://github.com/newrelic/newrelic-quickstarts.git
```

Or with an SSH origin
```bash
git remote set-url origin git@github.com:newrelic/newrelic-quickstarts.git
```

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
+
+<!-- DON'T DELETE THIS SECTION BELOW IF SUBMITTING A NEW QUICKSTART -->
+## Pre merge checklist
+
+<!-- This CHECKLIST SHOULD BE SUBMITTED FULLY COMPLETE WITH THE PR. IF NOT COMPLETE
+THE PR REVIEW WILL BE DELAYED -->
+
+-[ ] Did you check you NRQL syntax? - Does it work?
+-[ ] Did you check your dashboard image quality? -  Do they look good?
+-[ ] Did you checks that your alerts actually work?
```
Comment (nr-kkenney) [inline .github/pull_request_template.md:14]: typo here.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,8 +1,21 @@
-## Summary
+# Summary
 
-A concise description of the changes being introduced.
-If this is a new quickstart, please provide a description of its use cases and a list of the quickstart's components.
+A concise description of the changes being introduced. If you are submitting a PR for a
+new quickstart please review the Pre merge checklist section to validate this PR is ready for review.
```
Comment (aswanson-nr) [inline .github/pull_request_template.md:4]:
```suggestion
new quickstart please review the pre merge checklist section to validate this PR is ready for review.
```

File: .github/pull_request_template.md (hunk 1 above)
Comment (aswanson-nr) [reply .github/pull_request_template.md:14]:
```suggestion
-[ ] Did you check that your alerts actually work?
```

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Just two small typos! Content looks great

File: .github/pull_request_template.md (hunk 1 above)
Comment (jpvajda) [reply .github/pull_request_template.md]: That was in my smeagol voice.

File: quickstarts/dotnet/dotnet/config.yml [hunk 3]
```diff
@@ -3,35 +3,35 @@ name: dotnet
...
+
+  ### What Makes This quickstart Unique?
 
-  ## What Makes This quickstart Unique?
-  
-  With this quickstart, you can monitor health and status in one place, focus on the most important information, and enable preventative maintenance strategy. 
+  With this quickstart, you can monitor health and status in one place, focus on the most important information, and enable preventative maintenance strategy.
 summary: |
-  Learn more about.NET Framework, the importance of monitoring .NET, the ideal features of a .NET monitor, and the unique value of New Relic's .NET quickstart. 
+  Learn more about.NET Framework, the importance of monitoring .NET, the ideal features of a .NET monitor, and the unique value of New Relic's .NET quickstart.
```
Comment (nr-kkenney) [inline quickstarts/dotnet/dotnet/config.yml:34]:
```suggestion
  Learn more about .NET Framework, the importance of monitoring .NET, the ideal features of a .NET monitor, and the unique value of New Relic's .NET quickstart.
```

**Review by nr-kkenney — DISMISSED** [review]

**Review by aswanson-nr — DISMISSED** [review]

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**